core/test/presentation/views/text/rich_text_builder_test.dart

```diff
@@ -0,0 +1,280 @@
+import 'package:core/presentation/extensions/list_extensions.dart';
+import 'package:core/presentation/views/text/rich_text_builder.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  const normalTextStyle = TextStyle(
+    color: Colors.black,
+    fontSize: 16,
+  );
+  const highlightTextStyle = TextStyle(
+    color: Colors.red,
+    fontSize: 16,
+  );
+  Widget buildRichText({
```

Comment:
Should be private function

Reply:
Should I still make it private?

![Screenshot 2024-11-15 at 10 01 08](https://github.com/user-attachments/assets/cb0d5981-2304-45ec-bdca-e323579169f1)

Reply:
Oh, so keep it that way.